docs/source/software/conda.rst

```diff
@@ -16,14 +19,15 @@ Using a locally installed Conda virtual environment is highly recommended so tha
 You can also have more than one environment with different packages for testing purposes. This procedure uses the Anaconda module already loaded on Discovery.
 
 1. If you are on a login node, move to a compute node by typing ``srun --partition=short --nodes=1 --cpus-per-task=1 --pty /bin/bash``. In the above example, we request for 1 node with 1 cpu core, but you can request for additional resources as per your requirements.
-2. To check what version of Python you have installed, type ``which python``.
-3. To load anaconda, type ``module load anaconda3/2022.01``.
-4. To create your environment, type ``conda create -n <yourenvironmentname> python=3.7 anaconda``, where <yourenvironmentname> is the name you want to give your environment. Tip: to see a list of all of your conda environments, type ``conda info -e``. To save space, also use the ``--prefix=/work/<mygroup>/<mydirectory>`` flag to build in your work directory.
+2. To load anaconda, type ``module load anaconda3/2022.01``.
+3. To check what version of Python you have installed, type ``which python``. With the anaconda module loaded you should see the python directory as /shared/centos7/anaconda3/2022.01/bin/python. If you wish to use a different version of python you can load a python module. For example, ``module load python/3.8.1`` and then confirm that your python directory has changed with ``which python``.
+4. To create your environment in the /work or /scratch directory, first create a project folder for the environment using ``mkdir <yourenvironmentname>``. And now create the environment with ``conda create --prefix=/work/<mygroup>/<yourenvironmentname>``. You will see a message "WARNING: A directory already exists at the target location '/work/<mygroup>/<yourenvironmentname>' but is not a conda environment. Enter y to continue creating the environment. Tip: to see a list of all of your Conda environments, type ``conda info -e``.
 5. Follow the prompts to complete the Conda install.
-6. To activate your Conda environment, type ``source activate <yourenvironmentname>``. Note that ``conda activate`` will not work on Discovery with this version.
-7. To install a specific package, type ``conda install -n <yourenvironmentname> [package]``.
-8. To deactivate the current, active Conda environment, type ``conda deactivate``.
-9. To delete a Conda environment and all of its related packages, type ``conda remove -n <yourenvironmentname> --all``.
+6. To activate your Conda environment, type ``conda activate <yourenvironmentname>/``. Note, this command will only work with the ``/`` at the end of your environment name. You should now see the environment name on the left side of your username and current node information: (/work/group/username/test_condaRTD) [username@c2000 dirname]$
```

Comment:
I always have used source and not conda and I just tried it and it works so I think this is good. It might be worth mentioning you can also use source just for people who are used to using source and them knowing nothing changed and it will still work fine.